### ISSUE TITLE
[EPIC] Add Next of Kin to EZR 

### ISSUE THREAD

**Issue by alexseelig** [department-of-veterans-affairs/va.gov-team#91493]
##Initiative Brief

## Benefit to the Veteran
As a Veteran, I want [ ], so that [ ].
As a Veteran, I would like to be able to update my NoK online so that if something happens to me, VHA knows who to contact.

As a Veteran, I would like for the VA to be able to reach my next of kin via phone so that VHA can let my loved ones know if something happens to me.

As a Veteran, I would like the VA to know where my next of kin live so that they can contact them if something happens to me.

---
## Acceptance Criteria/Definition of Done
- [ ] View NoK
- [ ] Edit existing NoK
- [ ] Add NoK
- [ ] Remove existing NoK
- [ ] Cancel changes
- [ ] Same text fields as EC (name, address, #, relationship)
- [ ] NoK is Optional
- [ ] Part of Veteran info section, after EC
- [ ] Adding address is optional
- [ ] Saves in VES
- [ ] Only two allowed
- [ ] Pre-populate from existing data in VES
- [ ] Records added and removed on form submission, not on adding them to the form that has not yet been submitted

## Hypothesis or Bet
**If** _we make this change_ **then** _we expect this to happen_.
If we allow Veterans to update NoK online, we will improve data integrity within VHA systems.

If we allow Veterans to update NoK online, they will spend less time filling out paper forms.

If we allow Veterans to update NoK online, they will spend less time updating their information during check in/appointments.

If we allow Veterans to update their NoK online, the VA will have better data for them meaning the right person will be contacted in the event of something catastrophic. 

## OKR
_Which Objective / Key Result does this epic push forward?_
Digitizing forms
Allowing Veterans to do everything online that they can do with paper forms
Paperwork reduction for Veterans
Improved data quality within VHA systems

## Background
- _What details are necessary for understanding the specific work or request tracked by this issue?_
- This work is similar to, and in many ways part of a piece with, Emergency Contact. The fields will all be the same, as will the functionality. 

- _Is there anything that must be done before this ticket? (link ticket as dependency)_
- Ensure that VES can accept and save this data (link to story to check this)

- **In Scope:** 
1. View
2. Edit
3. Add
4. Delete
5. Cancel
6. Same text fields as EC (name, address, #, relationship)
7. Optional
8. Part of Veteran info section, after EC
10. Adding address is optional
11. Saves in VES
12. Only two allowed
13. Pre-populate from existing data in VES

**NOT in Scope**
1. "Same address as Veteran" checkbox--V2
2. Anything not explicitly listed above
3. More than two NoKs

**External Dependencies:**
1. VES can accept, save, and send back NoK information

**

## Resources
- Product or Initiative brief links 
- Design links
- Screenshots
- Reference docs/tickets where we have this kind of work documented
- Etc.

---

```[tasklist]
### Tasks (Team to Define)
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92427
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92260
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92325
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92419
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92420
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92421
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92426
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92425
- [ ] End to end testing
- [ ] Release Plan
- [ ] Update Source of Truth
- [ ] Team review & signoff
- [ ] Cypress test suite
- [ ] Accessibility Test
- [ ] Team review & signoff
- [ ] Figure out Delete
- [ ] Merge EC and NoK on BE?
- [ ] Investigate what happens with how schema behaves on frontend
- [ ] Launch to Prod
```

---
## How to configure this issue
- [ ] **Labeled with Team** (`1010-team`)
- [ ] **Labeled with Form** (`1010-ez`, `1010-ezr`, `1010-cg`)
- [ ] **Labeled with Project** (`facilities-cg`, `registration`,  `toxic-exposure`, etc.)

**Comment by alexseelig** [comment]
Apologies for the formatting wonkiness, working on that
